elasticdl/python/master/k8s_instance_manager.py

```diff
@@ -147,6 +206,29 @@ def __init__(
             self._num_ps, self._k8s_client.get_ps_service_address
         )
         self._worker_addrs = []
+        self._pod_event_callbacks = []
+        self._worker_command = None
+        self._worker_args = None
+        self._ps_command = None
+        self._ps_args = None
+
+    def set_up(
```

Comment:
Rename `set_up` to `set_pod_command` ?

Reply:
I prefer `set_up`. It means the set_up action which is necessary before start&run. Maybe we will add more parameters besides command arguments in this method.